_test/all-namespaces/ocp/bestpractices/all.yml

```diff
@@ -84,6 +84,16 @@ objects:
       app.kubernetes.io/managed-by: Bar
     name: shouldneverfire-${NAME}
   spec:
+    affinity:
+      podAntiAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: In
+                  values:
+                    - Foo
+            topologyKey: failure-domain.beta.kubernetes.io/zone
```

Comment:
Can `topology.kubernetes.io/zone` be used instead as the old key is deprecated

https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/#failure-domainbetakubernetesiozone

Reply:
done